trackData/frontend.json

```diff
@@ -61,6 +61,8 @@
         ],
```

Comment:
GitHub is being weird and doesn't let me add a comment/suggestion in the upper row.
I think we should add one more line in `"signals"`, something like `"Researches and suggests new tools for building better front end solutions by reducing the complexity and amount of code being written"`

Reply:
Also `"Has a good understanding of javascript bundlers and how to use them"`

Reply:
+1